src/app.js

```
@@ -1,77 +1,419 @@
+    // 错误处理问题：输入验证不完整
+    if (!username || !password) {
+        return res.status(400).json({ error: 'Missing credentials' });
+    }
+    
+    // 错误处理问题：密码哈希没有异常处理
+    const hashedPassword = crypto.createHash('md5').update(password).digest('hex');
+    
+    // 硬编码的查询语句
+    const query = 'SELECT * FROM users WHERE username = ? AND password = ?';
+    
+    db.execute(query, [username, hashedPassword], (err, results) => {
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(indent)